spec/lib/topics_bulk_action_spec.rb

```diff
@@ -159,6 +159,7 @@
         before { SiteSetting.create_revision_on_bulk_topic_moves = true }
 
         it "changes the category, creates a post revision and returns the topic_id" do
+          Group.refresh_automatic_groups!
```

Comment:
Instead of refreshing groups in each spec, you can create groups when user is fabricated (refresh_auto_groups: true). 
```
  fab!(:user) { Fabricate(:user, created_at: 1.days.ago, refresh_auto_groups: true) }
```